### PR TITLE
NRP-3051-Alert endpoints added

### PR DESCRIPTION
**Pull request by lgrzybowski**
There are endpoint for Alerts

Results for Rubocop:
`134 files inspected, no offenses detected`

and for rspec:
`Finished in 0.05699 seconds (files took 2.23 seconds to load)`

### CODE REVIEW

File: lib/bwapi/client/projects.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require 'bwapi/client/projects/alerts.rb'
```
Comment (jonathanchrisp) [inline lib/bwapi/client/projects.rb]: remove `.rb` - not required

File: lib/bwapi/client/projects/alerts.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        # @return [Hash] Alerts in project
+        def alerts(project_id)
+          get "projects/#{project_id}/alerts"
+        end
+
+        # Get a selected alert
+        #
+        # @param project_id [Integer] Id of project
+        # @param alert_id [Integer] Id of alert
+        # @return [Hash] Alerts in project
```
Comment (jonathanchrisp) [inline lib/bwapi/client/projects/alerts.rb]: `Alert in project`

File: lib/bwapi/client/projects/alerts.rb [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        def alerts(project_id, alert_id)
```
Comment (jonathanchrisp) [inline lib/bwapi/client/projects/alerts.rb]: this should be `get_alert` I believe, please compare against different endpoints

**Comment by jonathanchrisp** [comment]
@lgrzybowski - please can you fix my initial comments and then I'll re review

**Comment by lgrzybowski** [comment]
@jonathanchrisp Done:)

File: lib/bwapi/client/projects/alerts.rb (hunk 2 above)
Comment (jonathanchrisp) [inline lib/bwapi/client/projects/alerts.rb]: You haven't update the method comment, should read: `Alert in project`

**Comment by jonathanchrisp** [comment]
@lgrzybowski - sorry you forgot to do an update :cry: Please can you update and then its good to go :+1:

**Comment by lgrzybowski** [comment]
@jonathanchrisp Done :)

**Comment by jonathanchrisp** [comment]
LGTM :+1:

**Pull request merged**